2022-11-20 21:16:36,424 INFO    MainThread:17165 [wandb_setup.py:_flush():71] setting env: {}
2022-11-20 21:16:36,424 INFO    MainThread:17165 [wandb_setup.py:_flush():71] setting login settings: {}
2022-11-20 21:16:36,424 INFO    MainThread:17165 [wandb_init.py:_log_setup():371] Logging user logs to /home/josyula/Programs/MAS_Project/gym_envs_urdf/examples/wandb/run-20221120_211636-15ui1bky/logs/debug.log
2022-11-20 21:16:36,424 INFO    MainThread:17165 [wandb_init.py:_log_setup():372] Logging internal logs to /home/josyula/Programs/MAS_Project/gym_envs_urdf/examples/wandb/run-20221120_211636-15ui1bky/logs/debug-internal.log
2022-11-20 21:16:36,424 INFO    MainThread:17165 [wandb_init.py:init():404] calling init triggers
2022-11-20 21:16:36,424 INFO    MainThread:17165 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-11-20 21:16:36,424 INFO    MainThread:17165 [wandb_init.py:init():424] re-initializing run, found existing run on stack: 3l3oz4w6
2022-11-20 21:16:36,424 INFO    MainThread:17165 [wandb_run.py:finish():1457] finishing run josyula/nsra/3l3oz4w6
2022-11-20 21:16:36,443 INFO    MainThread:17165 [jupyter.py:save_history():429] not saving jupyter history
2022-11-20 21:16:36,443 INFO    MainThread:17165 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-11-20 21:16:36,443 INFO    MainThread:17165 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-11-20 21:16:36,443 INFO    MainThread:17165 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-11-20 21:16:36,444 INFO    MainThread:17165 [wandb_run.py:_restore():1752] restore
2022-11-20 21:16:38,106 INFO    MainThread:17165 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
}
pusher_stats {
}

2022-11-20 21:16:38,284 INFO    MainThread:17165 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 863
  total_bytes: 863
}

2022-11-20 21:16:39,121 INFO    MainThread:17165 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 863
  total_bytes: 863
}

2022-11-20 21:16:39,224 INFO    MainThread:17165 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 863
  total_bytes: 23005
}

2022-11-20 21:16:39,327 INFO    MainThread:17165 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 898
  total_bytes: 23005
}

2022-11-20 21:16:39,430 INFO    MainThread:17165 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 23005
  total_bytes: 23005
}

2022-11-20 21:16:39,533 INFO    MainThread:17165 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 23005
  total_bytes: 23005
}

2022-11-20 21:16:39,637 INFO    MainThread:17165 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 23005
  total_bytes: 23005
}

2022-11-20 21:16:39,740 INFO    MainThread:17165 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 23005
  total_bytes: 23005
}

2022-11-20 21:16:39,843 INFO    MainThread:17165 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 23005
  total_bytes: 23005
}

2022-11-20 21:16:40,143 INFO    MainThread:17165 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 23005
  total_bytes: 23005
}

2022-11-20 21:16:40,469 INFO    MainThread:17165 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 23005
  total_bytes: 23005
}
local_info {
}

2022-11-20 21:16:41,705 INFO    MainThread:17165 [wandb_run.py:_append_files():2180] logging synced files
2022-11-20 21:16:41,706 INFO    MainThread:17165 [wandb_init.py:init():460] starting backend
2022-11-20 21:16:41,706 INFO    MainThread:17165 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-11-20 21:16:41,708 INFO    MainThread:17165 [backend.py:ensure_launched():216] starting backend process...
2022-11-20 21:16:41,711 INFO    MainThread:17165 [backend.py:ensure_launched():221] started backend process with pid: 17301
2022-11-20 21:16:41,712 INFO    MainThread:17165 [wandb_init.py:init():469] backend started and connected
2022-11-20 21:16:41,719 INFO    MainThread:17165 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-11-20 21:16:41,719 INFO    MainThread:17165 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-11-20 21:16:41,719 INFO    MainThread:17165 [wandb_init.py:init():533] updated telemetry
2022-11-20 21:16:41,729 INFO    MainThread:17165 [wandb_init.py:init():563] communicating current version
2022-11-20 21:16:42,377 INFO    MainThread:17165 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.13.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-11-20 21:16:42,378 INFO    MainThread:17165 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-11-20 21:16:42,550 INFO    MainThread:17165 [wandb_init.py:init():606] starting run threads in backend
2022-11-20 21:16:47,553 INFO    MainThread:17165 [wandb_run.py:_console_start():1810] atexit reg
2022-11-20 21:16:47,554 INFO    MainThread:17165 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-11-20 21:16:47,554 INFO    MainThread:17165 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-11-20 21:16:47,555 INFO    MainThread:17165 [wandb_run.py:_redirect():1745] Redirects installed.
2022-11-20 21:16:47,555 INFO    MainThread:17165 [wandb_init.py:init():633] run started, returning control to user process
